src/handlers/warehouse.ts

```diff
@@ -37,12 +53,22 @@ export const warehouseHandlers: NatsHandler[] = [
             const operation = await update_item(warehouseContract, collectible);
 
             await operation.confirmation(1, 1);
+
+            client.publish(
+                replyTo,
+                `update_item is running, got ${collectible.item_id}`
```

Comment:
```suggestion
               collectible.item_id
```

let's just return the item_id here without the text around it, so the client doesn't have to parse the string to extract the relevant data. We can think of a more complex reponse with a human readable message as a next step